### DISCUSSION TITLE
Excluded from Hacktoberfesst 2024

### DISCUSSION THREAD

**Discussion by dhruvjain23** [Kushal997-das/Hacktoberfest-2021#174] [General]
I created a pull request in this repostiory , but on Hacktoberfest it shows that this repository has been excluded from hacktoberfest 2024 , reason is that it does not follow the values of the event.

**Comment by Kushal997-das** [comment]
Kindly take a SS and send it here.

**Comment by dhruvjain23** [comment]
![Screenshot 2024-10-03 180530](https://github.com/user-attachments/assets/e011d223-6ba3-46c3-a7cb-1c4b2c577f5d)
![Screenshot 2024-10-03 180543](https://github.com/user-attachments/assets/f9b53e1b-9af2-4534-a231-55531170d293)

**Comment by Kushal997-das** [comment]
I see kindly do contribute my other projects , specifically
Project-Guidance
